src/components/DimensionsPanel/Dialogs/AddToLayoutButton/__tests__/AddToLayoutButton.spec.js

```diff
@@ -14,7 +14,7 @@ describe('The AddToLayoutButton component ', () => {
     }
 
     // TODO remove
-    console.log('getShallowAddToLayoutButton', getShallowAddToLayoutButton)
+    getShallowAddToLayoutButton
```

Comment:
This looks strange. Is this function being called?

Reply:
Seems like line 16 and 17 should just be removed.

Reply:
I agree and tried that at first.

But this has a knock-on effect with ESLint. Removing the line causes an error about the `getShallowAddToLayoutButton` function being defined but never used (`no-unused-vars`), and then removing that function caused a bunch of other ESLint errors, and before you know it the whole file is empty 🤣 .

I am not sufficiently familiar with this part of the codebase to quickly asses if removing so much "we should implement this later" code can/should be removed.

So the current change does look odd but is least intrusive and doesn't involve us having to disable the linter. So on balance I thought it was the least bad solution.

Reply:
Wouldn't it be better to use // eslint-disable-next-line instead?

Reply:
OK, I'll fix it like that

Reply:
After some more deliberation we decided to just delete that whole file.